### DISCUSSION TITLE
Why Window.Current is null ?

### DISCUSSION THREAD

**Discussion by sigmarsson** [unoplatform/uno#5643] [Q&A]
At this moment, the desktop is about starting up. `InitializeComponent` not found, maybe the reason is the Wasm head does not compile? I do not know.

![image](https://user-images.githubusercontent.com/43340959/113270656-02a7a880-92da-11eb-981a-a0203dc184df.png)

This is the desktop csproj :

```
<Project Sdk="Microsoft.NET.Sdk">
	<PropertyGroup>
		<OutputType>WinExe</OutputType>
		<TargetFramework>net5.0-windows10.0.19041.0</TargetFramework>
		<TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
		<RootNamespace>Weather.History.WinUI</RootNamespace>
		<ApplicationManifest>app.manifest</ApplicationManifest>
		<Platforms>x86;x64;</Platforms>
		<RuntimeIdentifiers>win10-x86;win10-x64;</RuntimeIdentifiers>
	</PropertyGroup>
```

**Comment by Daoting** [comment]
Here's the doc: https://docs.microsoft.com/en-us/windows/apps/desktop/modernize/desktop-to-uwp-supported-api
